src/patternfly/components/Nav/nav.scss

```diff
@@ -1060,8 +1060,8 @@
   // Divider
   --pf-c-nav--c-divider--MarginBottom: 0;
 
-  & + & {
-    --pf-c-nav__section--MarginTop: var(--pf-c-nav__section--section--MarginTop);
+  & + & .pf-c-nav__section-title {
+    margin-top: var(--pf-c-nav__section--section--MarginTop); // update at breaking change
```

Comment:
This works, but can break someone who had `& + & { margin-top: [custom]; }` - they'll get a double margin. This is what has broken the openshift folks in the past. I think it's probably fine but nav isn't something we want to break. Do you think there is a different way?

Just an aside, this will be a great use of `:has()` when it lands!!

```
& + &:has(.pf-c-nav__section-title) {
  --pf-c-nav__section--MarginTop: var(--pf-c-nav__section--section--MarginTop);
}
```

Reply:
@mcoker updated